### PR TITLE
docs: onboarding papercuts

### PR DESCRIPTION
**Pull request by knickish**
Couple issues with getting set up that I encountered, broken links and such.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ To contribute to the development of xDSL follow the subsequent steps.
 git clone https://github.com/xdslproject/xdsl.git
 cd xdsl
 # set up the venv and install everything
-make venv
+make
```
Comment (knickish) [inline README.md:76]: `make venv` did not work for me on initial clone for some reason

Comment (superlopuh) [reply README.md:76]: CC @AntonLydike

**Review by superlopuh — APPROVED** [review]
Thank you for this!

File: README.md (hunk 1 above)
Comment (AntonLydike) [reply README.md:76]: Yes, I saw this yesterday, #2316 had a rename from `venv` to `venv/`, which broke the target silently, for now one needs to do `make venv/` to create the venv which is suboptimal.

Comment (AntonLydike) [reply README.md:76]: Opened #2419 to fix this!

Comment (superlopuh) [reply README.md:76]: @knickish  Once Anton's change is merged, could you please revert this part of the change, so we can merge the URL fixes?

**Review by webmiche — APPROVED** [review]
Nice, thanks a lot!

File: README.md (hunk 1 above)
Comment (knickish) [reply README.md:76]: sure, will do that

Comment (knickish) [reply README.md]: ok dropped that commit

**Comment by knickish** [comment]
whoops, guess merge is the way to go instead of rebase for these

**Pull request merged**